src/System.Private.Uri/src/System.Private.Uri.builds

```diff
@@ -10,14 +10,6 @@
     </Project>
     <Project Include="System.Private.Uri.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>netstandard13aot</TargetGroup>
-    </Project>
-    <Project Include="System.Private.Uri.csproj">
```

Comment:
Why are you removing the netstandard13aot build? If I recall correctly that was added for CoreRT support by @tarekgh. I suspect that build is definitely going to need to be live.